modules/QnA/resources/qnaService.js

```diff
@@ -584,12 +584,12 @@ DAL for Q&A Module
                     request['filter:updatedAtGreaterThanOrEqual'] = lastUpdatedAt;
                 }
 
-                // build list code cue point request
+                // build list code cue point request for qna settings (checking both new and old tags for BC)
                 var codeCuePointListRequest = {
                     'service': 'cuepoint_cuepoint',
                     'action': 'list',
                     'filter:entryIdEqual': entryId,
-                    'filter:tagsLike':'WEBCASTSTATETAG',
+                    'filter:tagsMultiLikeOr':'player-qna-settings-update,WEBCASTSTATETAG',
```

Comment:
@esakal since we need to support the old convention (up until both player and webcast app will get upgraded) I'm supporting both cases